config.xml

```diff
@@ -14,29 +18,18 @@
     <preference name="MediaPlaybackRequiresUserAction" value="false" />
     <preference name="UIWebViewBounce" value="false" />
     <preference name="DisallowOverscroll" value="true" />
-    <preference name="webviewbounce" value="false" />
     <preference name="BackupWebStorage" value="local" />
     <preference name="KeyboardDisplayRequiresUserAction" value="false" />
-    <preference name="SplashScreen" value="res/icon.png" />
-    <preference name="SplashScreenDelay" value="7000" />
-    <preference name="ShowSplashScreen" value="true" />
-    <preference name="SplashScreenWidth" value="600" />
-    <preference name="SplashScreenHeight" value="300" />
-    <preference name="SplashMaintainAspectRatio" value="true" />
-    <preference name="windows-target-version" value="10.0" />
-    <preference name="windows-phone-target-version" value="10.0" />
     <icon src="res/icon.png" />
     <platform name="android">
         <preference name="android-minSdkVersion" value="16" />
         <preference name="android-targetSdkVersion" value="23" />
-        <preference name="StatusBarOverlaysWebView" value="false" />
+        <preference name="BackgroundColor" value="0x00000000" />
```

Comment:
Should this use the same color as the default dark theme? It uses `#101010`.

Reply:
I was asking about that color to add it here, but this color only flashes for about a tenth of a second. When the app opens it starts at gray regardless, then shows this color for a split second, then moves to black for the main loading circle. Rather than have it flash three colors I just set it to match the third color since that one is shown the longest. No clue why any of that happens but this is the least jarring for now.

Reply:
:+1: That makes sense.